Can't build Dæmon with breakpad on Ubuntu 19.20 with GCC 9.2.1

```
Daemon/libs/breakpad/src/client/linux/handler/exception_handler.cc:109:12: error: ‘int tgkill(pid_t, pid_t, int)’ was declared ‘extern’ and later ‘static’ [-fpermissive]
  109 | static int tgkill(pid_t tgid, pid_t tid, int sig) {
      |            ^~~~~~
In file included from /usr/include/signal.h:374,
                 from Daemon/libs/breakpad/src/client/linux/handler/exception_handler.h:33,
                 from Daemon/libs/breakpad/src/client/linux/handler/exception_handler.cc:66:
/usr/include/x86_64-linux-gnu/bits/signal_ext.h:29:12: note: previous declaration of ‘int tgkill(__pid_t, __pid_t, int)’
   29 | extern int tgkill (__pid_t __tgid, __pid_t __tid, int __signal);
      |            ^~~~~~
```

--- a/src/client/linux/handler/exception_handler.cc
+++ b/src/client/linux/handler/exception_handler.cc
@@ -106,7 +106,7 @@
 #endif
 
 // A wrapper for the tgkill syscall: send a signal to a specific thread.
-static int tgkill(pid_t tgid, pid_t tid, int sig) {
+static int tgkill_wrapper(pid_t tgid, pid_t tid, int sig) {
   return syscall(__NR_tgkill, tgid, tid, sig);
   return 0;
 }
@@ -387,7 +387,7 @@ void ExceptionHandler::SignalHandler(int sig, siginfo_t* info, void* uc) {
     // In order to retrigger it, we have to queue a new signal by calling
     // kill() ourselves.  The special case (si_pid == 0 && sig == SIGABRT) is
     // due to the kernel sending a SIGABRT from a user request via SysRQ.
-    if (tgkill(getpid(), syscall(__NR_gettid), sig) < 0) {
+    if (tgkill_wrapper(getpid(), syscall(__NR_gettid), sig) < 0) {
       // If we failed to kill ourselves (e.g. because a sandbox disallows us
       // to do so), we instead resort to terminating our process. This will
       // result in an incorrect exit code.

